lib/doctest.js

```diff
@@ -66,9 +72,10 @@ module.exports = function(path, options) {
     return evaluate (options.module, source, path);
   } else {
     console.log ('running doctests in ' + path + '...');
-    var results = evaluate (options.module, source, path);
-    log (results);
-    return results;
+    return evaluate (options.module, source, path).then(function(results) {
+      log (results);
+      return results;
+    });
```

Comment:
The return value from `module.exports` has changed to Promise<results>, so this is a breaking change.